mmv1/third_party/terraform/services/cloudrunv2/data_source_google_cloud_run_v2_service_test.go

```diff
@@ -28,6 +33,8 @@ func TestAccDataSourceGoogleCloudRunV2Service_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("data.google_cloud_run_v2_service.hello", "id", id),
 					resource.TestCheckResourceAttr("data.google_cloud_run_v2_service.hello", "name", name),
 					resource.TestCheckResourceAttr("data.google_cloud_run_v2_service.hello", "location", location),
+					resource.TestCheckResourceAttr("data.google_cloud_run_v2_service.hello", "urls.#", "2"),
```

Comment:
given these are output fields, I don't believe they need to be verified in a test like this. is there a reason you added these I'm not realizing?

Reply:
According to https://googlecloudplatform.github.io/magic-modules/contribute/review-pr/: "all fields added/updated in the PR appear in at least one test". If this is not meant to apply to output only fields I wasn't able to find where this is stated.